docs/user/installation/local-repository.md

```diff
@@ -1,13 +1,15 @@
 # Install from local repository
 
-Make sure to clone the [tractus-x-umbrella](https://github.com/eclipse-tractusx/tractus-x-umbrella) repository beforehand and navigate to the `charts/umbrella` directory.
+Make sure to clone the [tractus-x-umbrella](https://github.com/eclipse-tractusx/tractus-x-umbrella) repository beforehand.
+
+Update the chart dependencies.
 ```bash
-cd charts/umbrella/
+./hack/helm-dependencies.bash
```

Comment:
In the install guide, I'm not a big fan of using the script in the hack directory, it's ok for workflows but here we wan't users to easily understand what's happening and using the script here makes it more hidden.